### ISSUE TITLE
四角

### ISSUE THREAD

**Issue by yuracream** [yuracream/link#3]
from keras.layers import Activation, Conv2D, Dense, Dropout, Flatten, MaxPooling2D
from keras.models import Sequential
from keras.utils import np_utils
import numpy as np
import cv2

# 学習データ
x = np.array([], np.int8)  # 画像データ
y = np.array([], np.int8)  # 正解データ

# 環境に合わせて変更してください。
dir_path = "c:/python/cnn/"
# 0       1
name_list = ['no_square', 'square']  # 四角がない、四角がある
for kind, kind_name in enumerate(name_list):
    for n in range(10):
        # 画像データを読み込んで、学習データとして追加します
        path = dir_path + kind_name + "/" + str(n) + ".png"
        img = cv2.imread(path, cv2.IMREAD_GRAYSCALE)  # グレースケールでファイル読み込み（縦 * 横 の2次元）
        x = np.append(x, img)  # 画像を1次元配列として追加
        y = np.append(y, kind)

# 画像データを4次元に変換します
x = x.reshape(-1, 28, 28, 1)

# 正解を one-hot 表現に変換
y = np_utils.to_categorical(y)

# モデルの生成
model = Sequential()

# 畳み込み層
model.add(Conv2D(filters=32, kernel_size=(3, 3), strides=(1, 1), padding='same', input_shape=(28, 28, 1)))
model.add(Activation('relu'))

# プーリング層
model.add(MaxPooling2D(pool_size=(2, 2)))

# 過学習の抑制
model.add(Dropout(rate=0.5))

# 全結合層へ変換
model.add(Flatten())

# 全結合層
